kani-driver/src/main.rs

```diff
@@ -194,6 +204,14 @@ impl KaniSession {
 
         Ok(())
     }
+
+    fn return_list_harnesses(self, sorted_harnesses: &[HarnessMetadata]) -> Result<()> {
```

Comment:
Can you please rename this function and remove the return type. Maybe call it something like `print_metadata`.